python-lib/language_dict.py

```diff
@@ -0,0 +1,272 @@
+    "an": "Aragonese",
+    "hy": "Armenian",
+    "as": "Assamese",
+    "az": "Azerbaijani",
+    "eu": "Basque",
+    "be": "Belarusian",
+    "bn": "Bengali",
+    "bs": "Bosnian",
+    "br": "Breton",
+    "bg": "Bulgarian",
+    "my": "Burmese",
+    "ca": "Catalan",
+    "km": "Central Khmer",
+    "zh": "Chinese",
+    "hr": "Croatian",
+    "cs": "Czech",
+    "da": "Danish",
+    "nl": "Dutch",
+    "dz": "Dzongkha",
+    "en": "English",
+    "eo": "Esperanto",
+    "et": "Estonian",
+    "fo": "Faroese",
+    "fi": "Finnish",
+    "fr": "French",
+    "gl": "Galician",
+    "ka": "Georgian",
+    "de": "German",
+    "el": "Greek",
+    "gu": "Gujarati",
+    "ht": "Haitian",
+    "ha": "Hausa",
+    "he": "Hebrew",
+    "hi": "Hindi",
+    "hu": "Hungarian",
+    "is": "Icelandic",
+    "ig": "Igbo",
+    "id": "Indonesian",
+    "ga": "Irish",
+    "it": "Italian",
+    "ja": "Japanese",
+    "jv": "Javanese",
+    "kn": "Kannada",
+    "kk": "Kazakh",
+    "rw": "Kinyarwanda",
+    "ky": "Kirghiz",
+    "ko": "Korean",
+    "ku": "Kurdish",
+    "lo": "Lao",
+    "la": "Latin",
+    "lv": "Latvian",
+    "lt": "Lithuanian",
+    "lb": "Luxembourgish",
+    "mk": "Macedonian",
+    "mg": "Malagasy",
+    "ms": "Malay",
+    "ml": "Malayalam",
+    "mt": "Maltese",
+    "mi": "Maori",
+    "mr": "Marathi",
+    "mn": "Mongolian",
+    "ne": "Nepali",
+    "se": "Northern Sami",
+    "nb": "Norwegian Bokmål",
+    "nn": "Norwegian Nynorsk",
+    "no": "Norwegian",
+    "ny": "Nyanja",
+    "oc": "Occitan",
+    "or": "Oriya",
+    "pa": "Panjabi",
+    "fa": "Persian",
+    "pl": "Polish",
+    "pt": "Portuguese",
+    "ps": "Pushto",
+    "qu": "Quechua",
+    "ro": "Romanian",
+    "ru": "Russian",
+    "sm": "Samoan",
+    "gd": "Scottish Gaelic",
+    "sr": "Serbian",
+    "sn": "Shona",
+    "sd": "Sindhi",
+    "si": "Sinhala",
+    "sk": "Slovak",
+    "sl": "Slovenian",
+    "so": "Somali",
+    "st": "Southern Sotho",
+    "es": "Spanish",
+    "su": "Sundanese",
+    "sw": "Swahili",
+    "sv": "Swedish",
+    "tl": "Tagalog",
+    "tg": "Tajik",
+    "ta": "Tamil",
+    "te": "Telugu",
+    "th": "Thai",
+    "tr": "Turkish",
+    "ug": "Uighur",
+    "uk": "Ukrainian",
+    "ur": "Urdu",
+    "uz": "Uzbek",
+    "vi": "Vietnamese",
+    "vo": "Volapük",
+    "wa": "Walloon",
+    "cy": "Welsh",
+    "fy": "Western Frisian",
+    "xh": "Xhosa",
+    "yi": "Yiddish",
+    "yo": "Yoruba",
+    "zu": "Zulu",
+}
+"""dict: Languages supported by pycld3
+
+Dictionary with ISO 639-1 language code (key) and language name (value)
+"""
+
+SUPPORTED_LANGUAGES_PYCLD3_NOT_LANGID = [
+    "fy",
+    "gd",
+    "ha",
+    "ig",
+    "mi",
+    "my",
+    "ny",
+    "sd",
+    "sm",
+    "sn",
+    "so",
+    "st",
+    "su",
+    "tg",
+    "uz",
+    "yi",
+    "yo",
+]
+"""list: Subset of languages codes supported by pycld3 but not langid"""
+
+LANGUAGE_REMAPPING_PYCLD3_LANGID = {"iw": "he", "co": "it", "ji": "yi", "in": "id"}
```

Comment:
No need for this one, it's used only for language detection

Reply:
done in 94885e4